plugins/woocommerce-admin/client/products/sections/pricing-section.tsx

```diff
@@ -243,6 +265,20 @@ export const PricingSection: React.FC = () => {
 								currencyConfig,
 								formatAmount
 							) }
+							placeholder={ getPlaceholderForVariation(
+								parent?.regular_price
+							) }
+							onFocus={ (
+								event: React.FocusEvent< HTMLInputElement >
+							) => {
+								setValueForVariation(
```

Comment:
Shouldn't we avoid setting the value on focus?  If the parent were to change, it would no longer follow the parent.

I think keeping this empty with a placeholder is what we want to do for these.

Reply:
> If the parent were to change, it would no longer follow the parent.

`setValueForVariation` considers that implementation.